primitives/session-keys/src/vrf.rs

```diff
@@ -17,28 +17,20 @@
 //! VRF Key type
 use nimbus_primitives::NimbusId;
 use sp_application_crypto::{sr25519, KeyTypeId, UncheckedFrom};
-use sp_consensus_babe::Transcript;
-#[cfg(feature = "std")]
-use sp_keystore::vrf::{VRFTranscriptData, VRFTranscriptValue};
+use sp_core::sr25519::vrf::{VrfInput, VrfSignData};
+//#[cfg(feature = "std")] <- TODO: Check if this is still needed
```

Comment:
TODO: check if we still need the `std` feature here.

To give more context, when we build the runtime with this line being uncommented, an error is thrown informing that `BoundToRuntimeAppPublic` type was not found in the scope. I guess it is because the runtime is making use of it within a non-std env.

Reply:
@librelois @nbaztec @notlesh any thoughts on this?

Reply:
This crate was moved to moonkit, but anyway we don't need this std feature, the types `BoundToRuntimeAppPublic` and `ConsensusEngineId` are not under std in polkadot.